pkg/shipper/shipper_test.go

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/thanos-io/thanos/blob/main/pkg/shipper/shipper_test.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Thanos Authors.
+
+package shipper
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"github.com/efficientgo/tools/core/pkg/testutil"
```

Comment:
Could you open an issue to get rid of this, and just use `testify` to keep consistency? We can do it separately.